tools/IoTEdgeDevOps/DevOpsLib/BugManagement.cs

```diff
@@ -73,6 +82,12 @@ public async Task<string> CreateBugAsync(string branch, VstsBuild build)
                         rel = "Hyperlink",
                         url = $"{build.WebUri}"
                     }
+                },
+                new
+                {
+                    op = "add",
+                    path = "/fields/System.AssignedTo",
+                    value = bugOwnerEmail
```

Comment:
If this email is empty string then the bug will be unassigned. This happens if we cannot resolve the committer name to someone on our team.